src/test/java/org/unicode/cldr/surveydriver/SurveyDriver.java

```diff
@@ -127,9 +128,6 @@ public static void main(String[] args) {
 		if (TEST_ANNOTATION_VOTING) {
 			s.testAnnotationVoting();
 		}
-		if (TEST_XML_UPLOADER) {
-			new SurveyDriverXMLUploader(s).testXMLUploader();
-		}
```

Comment:
I'll restore the deleted parts for TEST_VETTING_TABLE and TEST_XML_UPLOADER; I deleted them while testing a minimal test version without those two files